Dockerfile.rhel

```diff
@@ -12,47 +12,45 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM registry.access.redhat.com/rhel7/rhel
+FROM registry.access.redhat.com/ubi8/ubi
```

Comment:
this makes me wonder in my own PR for docker-nexus3, if I needed to retain the rhel images, or if I really only need ubi.  I kept ubi separate.

Reply:
since there *is* a rhel platform, it might be clearer to rename this file Dockerfile.rh.ubi or something.

Reply:
> this makes me wonder in my own PR for docker-nexus3, if I needed to retain the rhel images, or if I really only need ubi. I kept ubi separate.

I'm not sure of the value of the rhel7 image. I don't think anyone outside of the RHCC would use it, so I'm going to skip it until there's a demonstrated need, otherwise we might be stuck maintaining it. (Plus I couldn't get it running)

Reply:
> since there _is_ a rhel platform we used to use, it might be clearer to rename this file Dockerfile.rh.ubi or something.

I don't see so much value in the `.ubi` since both images are from `ubi`, but I like `.rh` (dropping the `el` which might be confusing). Is that okay with you?